genie-client/src/main/java/com/netflix/genie/client/apis/JobService.java

```diff
@@ -112,7 +115,9 @@
         @Query("minFinished") Long minFinished,
         @Query("maxFinished") Long maxFinished,
         @Query("grouping") String grouping,
-        @Query("groupingInstance") String groupingInstance
+        @Query("groupingInstance") String groupingInstance,
+        @Query("page") Integer page,
+        @Query("size") Integer size
```

Comment:
Any particular reason for not including the other 2 parameters (order/direction)